package.json

```
@@ -1,10 +1,11 @@
 {
-  "name": "react-mdb-template",
-  "version": "1.0.2",
+  "name": "snek-front",
+  "version": "1.0.0",
```

Wrong version number. Change it to:
```suggestion
  "version": "0.1.0",
```